autoload/codi/load.vim

```diff
@@ -118,21 +118,24 @@ endfunction
 
 let s:codi_default_interpreters = {
       \ 'python': {
-          \ 'bin': ['env', 'PYTHONSTARTUP=', 'python'],
-          \ 'prompt': '^\(>>>\|\.\.\.\) ',
+          \ 'bin': has('win32') ? ['python']
+          \    : ['env', 'PYTHONSTARTUP=', 'python'],
+          \ 'prompt': '^\(>>>\|\.\.\.\) \=',
```

Comment:
Instead of adding ` \=` to each interpreter, what do you think of adding it to the usage sites instead, so new interpreters work on Windows without remembering to add `\=`?

Reply:
Please let me know if this is what you meant moving it to the usage sites

Reply:
Yup, exactly - thank you!